Portfolio Setup and Mobile First: Add index.html, style.css and Linters

rmiyoyo opened the pull request:
# Work Done 
- Added the ```index.html```, ```style.css``` files and images for the portfolio site
- Added the HTML/CSS linters to check for errors.
- Implemented flexbox

rmiyoyo committed:
Add a customize Microverse README template

rmiyoyo committed:
Add index.html and style.css files that contain the portfolio website

rmiyoyo committed:
Add HTML/CSS linters config files

rmiyoyo committed:
Fix template errors by removing Hello-Microverse name

rmiyoyo committed:
Fix linter errors in style.css file

rmiyoyo committed:
Add correct screenshot to README

OyePriscilla commented on index.html:
```diff
@@ -0,0 +1,41 @@
+<body>
+  <div class="container">
+    <nav>
+      <a href="#" class="logo">My Logo</a>
```
- [ ] Please kindly bring the logo out of the `nav` tag, a logo is not supposed to be part of the navigation links. 👍

OyePriscilla commented on index.html:
```diff
@@ -0,0 +1,41 @@
+  <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Poppins:wght@100;400;600;700&display=swap">
+</head>
+<body>
+  <div class="container">
```
- [ ] Please for semantics purpose, kindly use the `header` tag in place of the div tag. The [HTML](https://developer.mozilla.org/en-US/docs/Web/HTML) header element represents introductory content, typically a group of introductory or navigational aids. 👍

OyePriscilla commented on index.html:
```diff
@@ -0,0 +1,41 @@
+      <h1>I'm Raphael.<br /> I am so glad to see you!</h1>
+    </div>
+  
+    <div class="main">
+      <p>I'm a software developer! I can help you build a product, feature, or website. Look through some of my work and experience! If you like what you see and have a project you need coded, don't hesitate to contact me.</p>
+    </div>
+  
+    <div class="connect text-left">
+      <p class="social-media">LET'S CONNECT</p>
+      <div class="social-icons">
+        <a href="#"><img src="./images/icons/twitter.svg" alt="Twitter"></a>
+        <a href="#"><img src="./images/icons/Linkedin.png" alt="LinkedIn"></a>
+        <a href="#"><img src="./images/icons/m.svg" alt="Medium"></a>
+        <a href="#"><img src="./images/icons/github.svg" alt="GitHub"></a>
+        <a href="#"><img src="./images/icons/random.svg" alt="Random"></a>
+      </div>
+    </div>
+  </div>
```
- [ ]  The container of this section suppose to be a `section` not `div` tag. It represents a generic standalone section of the document. 👍

OyePriscilla commented on index.html:
```diff
@@ -0,0 +1,41 @@
+    <div class="connect text-left">
+      <p class="social-media">LET'S CONNECT</p>
+      <div class="social-icons">
+        <a href="#"><img src="./images/icons/twitter.svg" alt="Twitter"></a>
+        <a href="#"><img src="./images/icons/Linkedin.png" alt="LinkedIn"></a>
+        <a href="#"><img src="./images/icons/m.svg" alt="Medium"></a>
+        <a href="#"><img src="./images/icons/github.svg" alt="GitHub"></a>
+        <a href="#"><img src="./images/icons/random.svg" alt="Random"></a>
```
- [ ]  Kindly use a `ul` tag to group the list of these images, and `li` per icon; each icon has a link (a tag). Apart from being semantically correct, the a tag will let you connect your social networks with the icons, using `href`. 👍

OyePriscilla commented on style.css:
```diff
@@ -0,0 +1,116 @@
+  --text-color: #344563;
+}
+
+body {
+  margin: 0;
+  background-image: url("./images/bg.png");
+  background-repeat: no-repeat;
+  background-position: center 76px;
+  background-size: cover;
+  font-family: 'Poppins', sans-serif;
+}
```
- [ ] Please kindly take note of the general structure of your design when compared to the given template. I couldn't see the `left-border-bottom` radius and more so, all the background images are not showing. Please ensure to align all the elements on the page according to the given template. Our ability to reproduce perfectly a given design shows that we pay attention to details. 👍 

## Figma design
<img width="141" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227379198-c625d3b2-8781-49da-8260-f2d7812506bf.png">

## Your design
<img width="300" alt="Untitled" src="https://user-images.githubusercontent.com/103386183/227379363-122774bc-a76f-486c-a3aa-2fe56f505c01.png">

OyePriscilla reviewed (changes requested):
Hi @rmiyoyo 👍 , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

![so-close](https://user-images.githubusercontent.com/103386183/196423863-0f522555-1b03-455e-a452-b822cdcfc398.gif)

## Highlights
Linters check passed ✔️
Observed GitHub Flow ✔️
Content is aligned with flexbox ✔️
Used images and background-image to enhance the look of the website ✔️

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

rmiyoyo committed:
Replace screenshot with updated website

rmiyoyo committed:
Add section, remove logo from nav and update background

rmiyoyo committed:
Add new screenshot of website

rmiyoyo committed:
Fix stylint issues in style.css

rmiyoyo commented:
# Work Done After Review
- Removed logo from ```nav``` tag
- Adjusted the alignment of sections to be uniform
- Added ```header``` tag after the main container ```div```
- Replaced ```div``` with ```section``` tag in the social media icons
- Used an unordered list in the social media section
- Revised general structure of the design to include all background images and alignments

Shedrack-Sunday reviewed (approved):
# STATUS: APPROVED ♻️♻️
Hi @rmiyoyo 

To highlight 👍🏽
✔️ Proper use Semantics
✔️ No linter errors

Your project is complete! There is nothing else to say other than... it's time to merge it 🚀
Congratulations! 🎉

## Optional suggestions
Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.
